CONTRIBUTING.md

```diff
@@ -89,6 +89,42 @@ After build completes, you can preview the resulting site locally using built-in
 	
 This will start the local preview on localhost:8080. You can then view the changes by going to http://localhost:8080/docs/[path] (e.g. http://localhost:8080/docs/getting-started/).   
 
-## Contributing to samples
+# Contributing to samples
 
-See the [Samples Readme](https://github.com/dotnet/core-docs/blob/master/samples/README.md).
+We encourage samples for any of our topics. Readers can download and run the samples. They
```

Comment:
I prefer to avoid ownership-related words, like "our". How about changing the framing a bit ...

> Readers would love to see samples for all topics in .NET documentation. They like to run them and explore the code and experiment with their own changes. Contributions that add new samples are considered high value and are likely to get accepted.